app/serializers/v1/keyword_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module V1
+  class KeywordSerializer
+    include JSONAPI::Serializer
```

Comment:
It would be best to create a base API controller and make that class inherit from it not to have to include that module in each serializer 💡

Reply:
Fixed in b55c8f986c755e4cfa50f22f26e118f1474428fb